qt/python/mantidqt/plotting/figuretype.py

```diff
@@ -29,14 +31,18 @@ class FigureType(Enum):
     Line = 1
     # Line plot with error bars
     Errorbar = 2
+    # A waterfall plot
+    Waterfall = 3
     # An image plot from imshow, pcolor, pcolormesh
-    Image = 3
+    Image = 4
     # A 3D surface plot
-    Surface = 4
+    Surface = 5
     # A 3D wireframe plot
-    Wireframe = 5
+    Wireframe = 6
     # A contour plot
-    Contour = 6
+    Contour = 7
+    # A colorfill plot
+    Colorfill = 8
```

Comment:
Since this is not used anymore can we remove it to avoid confusion in the future